test-app/runtime/src/main/java/com/tns/Runtime.java

```diff
@@ -274,6 +274,14 @@ private static String getStackTraceOnly(String content) {
         return result;
     }
 
+    public static String getJSStackTrace(Throwable ex) {
+        if (ex instanceof NativeScriptException) {
+            return ((NativeScriptException) ex).getIncomingStackTrace();
+        } else {
+            return null;
```

Comment:
It would be preferable to return an empty Optional<String> in this case in order to mitigate future NullPointerExceptions